### ISSUE TITLE
Allow changing the label markup

### ISSUE THREAD

**Issue by zerolab** [torchbox/tbxforms#68]
While tbxforms marks optional labels with "(required)", we had requests to stick to the asterisk denoting required fields.

At the moment that is not possible unless one overrides the whole `field.html` or use JS.

Can we:

- add a required class to labels (e.g. `tbxforms-label--required`)
- allow passing additional classes to labels, or the wrapping container
- optionally have TBXFORMS_REQUIRED_LABEL_MARKER settings, defaulting to an empty string?

**Comment by siimonevans** [comment]
Yes please! Without this I've found I've had to write gnarly CSS (and/or JS) which feels unnecessary.

**Comment by kbayliss** [comment]
Solved by https://github.com/torchbox/tbxforms/pull/78
